org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/SemanticTokenModifiers.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2016-2018 TypeFox and others.
```

Comment:
Please adapt the year and main copyright holder (put your name or affiliation).

Reply:
Same for other new files.

Reply:
Added for the 3 new files that are not generated by `xtend`